Emby.Server.Implementations/ScheduledTasks/Tasks/DeleteTranscodeFileTask.cs

```diff
@@ -52,14 +47,7 @@ public Task Execute(CancellationToken cancellationToken, IProgress<double> progr
             var minDateModified = DateTime.UtcNow.AddDays(-1);
             progress.Report(50);
 
-            try
-            {
-                DeleteTempFilesFromDirectory(cancellationToken, ApplicationPaths.TranscodePath, minDateModified, progress);
-            }
-            catch (DirectoryNotFoundException)
-            {
-                // No biggie here. Nothing to delete
-            }
+            DeleteTempFilesFromDirectory(cancellationToken, _configurationManager.GetTranscodingTempPath(), minDateModified, progress);
```

Comment:
why?

Reply:
We check if it exists first, and ignoring exceptions is bad practice.